content/rdi/installation/install-k8s.md

```diff
@@ -21,7 +21,7 @@ Throughout this document, the snippets make use of the Kubernetes `kubectl` tool
 > Note: The version of RedisGears you will install must match the base OS of the Redis Enterprise containers. In case of [Rancher](https://www.rancher.com/), the Redis Enterprise container base OS is Ubuntu 18.04. Use the following command to install RedisGears:
 
 ```bash
-curl -s https://redismodules.s3.amazonaws.com/redisgears/redisgears.Linux-ubuntu18.04-x86_64.1.2.6-withdeps.zip -o /tmp/redis-gears.zip
+curl -s https://redismodules.s3.amazonaws.com/redisgears/redisgears.Linux-ubuntu18.04-x86_64.1.2.7-withdeps.zip -o /tmp/redis-gears.zip
```

Comment:
We should use the variable in `config.yaml` instead of putting `1.2.7` here hardcoded.

Reply:
✅